website/static/js/folderPickerNodeConfig.js

```diff
@@ -510,7 +510,21 @@ var FolderPickerViewModel = oop.defclass({
                     custom : FolderPicker.selectView
                 }
             ];
-            }
+            },
+            xhrconfig : function(xhr) {
+                xhr.withCredentials = true;
+            },
+            lazyLoadPreprocess: function(data) {
+                // Also handle data from API -- squash `attributes` to what TB expects
+                if (data.data) {
+                    for (var i = 0; i < data.data.length; i++) {
+                        $.each(data.data[i].attributes, function(key, value) {
```

Comment:
Per in person conversation: Tom Baxter mentions a possible solution to this problem in candidate PR #5240 .